argus/action_manager/base.py

```diff
@@ -77,13 +77,17 @@ def check_cbinit_service(self, searched_paths=None):
         pass
 
     @abc.abstractmethod
-    def git_clone(self, repo_url, location):
-        """Clone from an remote repo to a specific location on the instance.
-
-        :param repo_url:
-            The remote repo url.
-        :param location:
-            Specific location on the instance.
+    def git_clone(self, repo_url, location, count, delay):
+        """Clone from a remote repository to a specified location.
```

Comment:
`specified location` refers to the location parameter specified by the user.Don't see how that isn't clear enough.